presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -81,7 +81,7 @@ public BlackHoleMetadata()
     public synchronized void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties)
     {
         if (schemas.contains(schemaName)) {
-            throw new PrestoException(ALREADY_EXISTS, String.format("Schema [%s] already exists", schemaName));
+            throw new PrestoException(ALREADY_EXISTS, format("Schema [%s] already exists", schemaName));
```

Comment:
Dear Reviewer,

don't get misled. This is just code cleanup. The first commit is what matters.

Reply:
We should squash these commits together, as the first one leaves this as an unnecessarily qualified method call. It was necessary until the import was changed, so the first commit is creating this code to be cleaned up, thus they should be together.

Reply:
Generally I agree. In this case, however, this would be confusing, effectively obfuscating what the change was about.